distributed/cli/utils.py

```diff
@@ -69,3 +71,12 @@ def uri_from_host_port(host_arg, port_arg, default_port):
     addr = unparse_address(scheme, loc)
 
     return addr
+
+
+def create_ssl_context(certfile, keyfile):
+    if certfile and keyfile:
+        ssl_ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
```

Comment:
The `ssl` module chooses slightly different parameters for servers and clients. Right now the differences may not matter much, but they can start diverging in the future.
At least there should be a comment here about this possible gotcha.